feed/index.html

```diff
@@ -17,9 +17,9 @@
               <ul class="navbar-nav mt-4">
```

Comment:
Maybe put this in a <header> or a <nav> container? Not necessary though, just my two cents 🙂